js/app/dashboard/components/TaskGroup.js

```diff
@@ -125,7 +133,7 @@ class TaskGroup extends React.Component {
             { this.state.editName && this.renderEditNameForm() }
           </h4>
         </div>
-        <div id={ `task-group-panel-${this.state.group.id}` } className="panel-collapse collapse" role="tabpanel">
+        <div id={ `task-group-panel-${this.state.group.id}` } className={classNames({"panel-collapse": true,  "collapse": true, "in": isExpanded})} role="tabpanel">
```

Comment:
if a css class is always set, it can be also written like `classNames("panel-collapse collapse", {"in": isExpanded})`

Reply:
cool thx i didnt know